db/schema.rb

```diff
@@ -349,6 +349,7 @@
     t.integer "ukprn"
     t.text "fe_type"
     t.boolean "hide_mno", default: false
+    t.datetime "opted_out_at"
```

Comment:
Sorry to be a pain, but do you think we could rename this to something a bit more descriptive? `opted_out_of_communications` ?

Reply:
I've changed to `opted_out_of_comms_at` as it's a timestamp not a bool and a bit shorter.
No pain, that's the point of review 🙂